src/Uno.UI.WpfHost/Uno.UI.WpfHost.csproj

```diff
@@ -87,7 +87,7 @@
       <Version>3.3396.1786</Version>
     </PackageReference>
     <PackageReference Include="cef.redist.x86">
-      <Version>3.3396.1786</Version>
```

Comment:
This will mean consumers of the `Uno.UI.WpfHost` nuget package will need to use at min v73.1.130 of CEF. I think always forcing consumers to upgrade to the latest of CEF is merited as CEF is web-browser and updates will typically include security fixes. Thoughts?